modules/mono/editor/bindings_generator.cpp

```diff
@@ -387,7 +387,7 @@ String BindingsGenerator::bbcode_to_xml(const String &p_bbcode, const TypeInterf
 					xml_output.append(link_target);
 					xml_output.append("</c>");
 				}
-			} else if (link_tag == "const") {
+			} else if (link_tag == "constant") {
```

Comment:
@neikeq

Reply:
I just checked and there's no `[const ` occurrences. It's `constant` indeed.